soccer/gameplay/tactics/wall.py

```diff
@@ -102,11 +102,13 @@ def execute_defense_wall(self):
             self._remove_wall_defenders()
             self.active_defenders = self.number_of_defenders
 
-    ## Returns true if some team has possession of the ball
-    def is_ball_not_free(self):
-        return main.ball().vel.mag() > 1 or min([(main.ball().pos - rob.pos).mag() \
-            for rob in main.system_state().their_robots]) <= min([(main.ball().pos - rob.pos).mag() \
-                for rob in main.system_state().our_robots])
+    ## Returns true if no team has possession of the ball
+    def is_ball_free(self):
```

Comment:
To be more specific, the ball is not moving fast and we are closer than the opponents to the ball